TensorMath.lua

```diff
@@ -96,6 +96,8 @@ wrap.types.CudaTensor = {
       local txt = {}
       if arg.creturned then
          -- this next line is actually debatable
+         -- doesn't this guarantee a leak in some cases?
+         -- I suggest we assert against creturned tensors...
```

Comment:
Good point. I guess it's not really clear whether the function that returns the tensor keeps a reference to it or not. Seems like this isn't used at the moment, so maybe we should just assert against it, as you suggested.